server/pulp/server/webservices/urls.py

```diff
@@ -57,6 +57,14 @@
         name='content_orphan_type_subcollection'),
     url(r'^v2/content/orphans/(?P<content_type>[^/]+)/(?P<unit_id>[^/]+)/$',
         OrphanResourceView.as_view(), name='content_orphan_resource'),
+    url(r'^v2/content/sources/$', ContentSourceView.as_view(),
+        name='content_sources'),
+    url(r'^v2/content/sources/action/(?P<action>[^/]+)/$', ContentSourceView.as_view(),
+        name='content_sources_action'),
+    url(r'^v2/content/sources/(?P<source_id>[^/]+)/action/(?P<action>[^/]+)/$', ContentSourceResourceView.as_view(),
```

Comment:
Could you move the View to the next line? this isn't getting tested by flake, but it is past the github window.